nixos/modules/misc/documentation.nix

```
@@ -357,6 +357,14 @@ in
     (mkIf cfg.nixos.enable {
       system.build.manual = manual;
 
+      system.activationScripts.check-manual-docbook = ''
+        if [[ $(cat ${manual.optionsUsedDocbook}) = 1 ]]; then
+          echo -e "\e[31mwarning\e[0m: This configuration contains option documentation in docbook." \
```

```suggestion
          echo -e "\e[31;1mwarning\e[0m: This configuration contains option documentation in docbook." \
```

Change from red to bold red, just like `lib.warn`.